docs/get-started/quickstart.md

```diff
@@ -25,6 +25,33 @@ What you'll learn:
 
 You'll need a Sumo Logic account. Sign up for a free trial [here](/docs/get-started/sign-up).
 
+## Getting started with kickstart data in your trial
+
+With your [trial](/docs/get-started/sign-up), you can access preloaded kickstart placeholder data, letting you explore Sumo Logic without setting up your own data. This feature helps you experience the platform’s value immediately, bypassing setup barriers like firewall and security configurations.
+
+### Key benefits
+
+* **Immediate insights**. Explore Sumo Logic right away, without initial data setup, to quickly see its value and decide if it’s a fit for you.
+* **Quick setup**. Placeholder data removes technical hurdles, making onboarding faster and easier.
+* **Guided experience**. Access pre-built dashboards and reports that demonstrate real-world scenarios, helping you make the most of your trial.
+* **Safe exploration**. Evaluate our platform in a secure environment with no exposure of sensitive data.
+
+### How it works
+
+1. **User assignment**. When you start your trial, you will be automatically assigned to either the placeholder data workflow or the custom data workflow, based on your user profile.
+2. **Preloaded sample data**. If you are assigned the placeholder data option, sample data will be preloaded into the platform, along with dashboards and log searches tailored to your use cases, such as Monitoring & Troubleshooting (M&T) or Compliance & Security (CIS).
```

Comment:
@kimsauce, I'd remove this phrase: 
", such as Monitoring & Troubleshooting (M&T) or Compliance & Security (CIS)" 

I'm not sure those terms are correct. In Sumo Logic, "CIS" stands for "Cloud Information Security", not "Compliance & Security".